packages/nextjs/components/scaffold-eth/Contract/InputUI.tsx

```diff
@@ -60,6 +60,7 @@ const InputUI = ({ setForm, form, stateObjectKey, paramType }: TInputUIProps) =>
               formUpdate[event.target.name] = contractFunctionArgument;
               setForm(formUpdate);
             }}
+            type={paramType.type === "uint256" ? "number" : undefined}
```

Comment:
do we need that arrows?

<img width="584" alt="Screenshot 2023-02-25 at 19 14 40" src="https://user-images.githubusercontent.com/25638585/221364783-1d05d0d4-67a8-4704-8363-d0b691d54d69.png">

Reply:
++ vote to remove it